tests/base/value_error.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+import os
+import yaml
+import json
+from ._base import BaseTestClass
+
+
+class BaseValueErrorTest(BaseTestClass):
+
+    def pytest_generate_tests(self, metafunc):
+
+        invalid_files = self.invalid_files()
+
+        parametrized = []
+
+        for file in invalid_files:
+            parametrized.append((file, file))
```

Comment:
No big deal here but I would not use `file` as a variable. It will overwrite the global file object.